tools/qontract_cli.py

```diff
@@ -517,6 +517,15 @@ def get_available_upgrades(ocm: OCM, version: str, channel: str) -> list[str]:
     return [u for u in upgrades if not ocm.version_blocked(u)]
 
 
+def inheritVersionDataText(ocm_org: str, ocm_specs: list[dict]) -> str:
+    ocm_spec = [o for o in ocm_specs if o["name"] == ocm_org][0]
```

Comment:
this PR is nice!

so allow me some nits-picking :D 

is there a situation where `org_name` is not available in the `ocm_specs`? in the current call contexts for sure not because you control that well. but if this method would be called from somewhere else, the access to [0] would fail.

you could use next with an iterator, resolve to `None` if not available and explicitly check for it.
```
    ocm_spec = next(o for o in ocm_specs if o["name"] == ocm_org], None)
    if ocm_spec:
       ...
```
the nice thing about `next` is, that it stops the moment you find an element. for sure not relevant in this context but hey, this is about nit-picking :)

Reply:
TIL!
I am not against it, but I am wondering if that's better though. Shouldn't we always call that function knowing the org with that name exists ? Isn't it better to fail in that case?

Reply:
failing is a good thing. but it should be something nicer than an `IndexError`. so it is absolutely fine to not use the next/iterator pattern. checking for len and failing with a generic `ValueError` with a readable error message works as well.

Reply:
fixed in [ad2f6db](https://github.com/app-sre/qontract-reconcile/pull/3236/commits/ad2f6dbb707af5b396c302549ad0e36e9e6695af)